src/Tizen.NUI/src/public/BaseComponents/RiveAnimationView.cs

```diff
@@ -0,0 +1,164 @@
+/*
+* Copyright(c) 2021 Samsung Electronics Co., Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// RiveAnimationView renders an animated vector image (Rive file).
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class RiveAnimationView : View
+    {
+        static RiveAnimationView() { }
+
+        /// <summary>
+        /// RiveAnimationView constructor.
+        /// </summary>
+        /// <param name="url">The rive resource URL</param>
+        ///[EditorBrowsable(EditorBrowsableState.Never)]
+        public RiveAnimationView(string url) : this(Interop.RiveAnimationView.New(url), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        internal RiveAnimationView(global::System.IntPtr cPtr, bool shown = true) : base(cPtr, shown)
+        {
+            if (!shown)
+            {
+                SetVisible(false);
+            }
+        }
+
+        /// <summary>
+        /// Eanables the given animation.
+        /// </summary>
+        /// <param name="animationName">The animation to enable</param>
+        /// <param name="on">The enable state of given animation</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void EnableAnimation(string animationName, bool on)
+        {
+            Interop.RiveAnimationView.EnableAnimation(SwigCPtr, animationName, on);
+        }
+
+        /// <summary>
+        /// Play animation.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void PlayAnimation()
```

Comment:
How about change the name as just `Play()` ?
for example, `LottieAnmationView` and `Animation` classes have the names of `Play()`, `Pause()` and `Resume()`.